source/api/auth/0.9/index.md

```diff
@@ -46,17 +46,23 @@ This is a work in progress. We are actively seeking implementations and feedback
 
 The IIIF (pronounced "Triple-Eye-Eff") specifications are designed to support uniform and rich access to resources hosted around the world. Open access to content is desirable, but policy, regulations, or business models can require users to authenticate and be authorized to interact with access-controlled resources. The authentication process could range from a simple click-through agreement to a multi-factor scheme with a secure identity provider.
 
-Content providers often offer tiers of access beyond a simple all-or-nothing distinction. These tiers could provide versions of a  resource that differ based on resolution, watermarking, or compression. Each version must have a distinct URI to prevent web caches from providing the wrong version. 
+Content providers often offer tiers of access beyond a simple all-or-nothing distinction. These tiers could provide versions of a resource that differ based on resolution, watermarking, or compression. Each version must have a distinct URI to prevent web caches from providing the wrong version.
 
 Providing interoperable content through client applications running in a web browser poses many challenges:
 
 * A single IIIF Presentation API manifest can reference content resources at multiple institutions and hence from multiple domains.
-* Institutions have different existing access control systems and should not have to adopt a new one to participate.
-* A IIIF client can be a JavaScript viewer served from a different domain from the image services, and the authentication services that protect them. This domain is _untrusted_ - the authorizing server must not require any prior knowledge of the domain hosting the viewer. The specification aims to not introduce or require any registry of trusted IIIF viewer domains and must assume that for image delivery, anyone can create any kind of viewer and run it from anywhere.  
+* Institutions have different existing access control systems.
+* Most IIIF viewers are client-side JavaScript applications, and may be served from a domain that is different, and thus untrusted, from the image services that it is required to load.
+ * Similarly, the domain of the authentication services that protect IIIF-based services may be different from that of a viewer or IIIF resources. Therefore, the authorizing server must not require any prior knowledge of the domain hosting the viewer.
+
+Additionally, the IIIF community has the following goals for this specification:
+
 * A IIIF client should not ask for or accept any credentials itself; the server hosting the content must be responsible for capturing credentials from a user and the IIIF viewer needs no knowledge of or access to this exchange.
 * A browser-based IIIF client must be able to maintain its state during an authentication flow.
+* No registry of trusted domains is required. Anyone should be able to create any kind of viewer and run it from anywhere.
```

Comment:
A registry of trusted domains should not be required.  (to follow the pattern of the other bullets)